### ISSUE TITLE
7.15.3. Screen Name/Title: New features - Links Lack Descriptive Context

### ISSUE THREAD

**Issue by jzbahrai** [cds-snc/notification-planning#2280]
Components: Specification
Observation: 
Links Lack Descriptive Context
Links like “Specification”, “contact us”, or “Accessibility Statement” lack unique context for screen reader users.

(Attachment: 
7.15.3. Links Lack Descriptive Context.mp4)

Recommendation: 
Ensure links provide more descriptive link text, or include visually hidden text to clarify (e.g., "Read the OpenAPI Specification documentation").

**Comment by YedidaZalik** [comment]
Links have been removed in both EN and FR
